releasenotes/notes/add-w3c-traceparent-propagation-789324b2e1326f0e.yaml

```
@@ -0,0 +1,8 @@
+---
+features:
+  - |
+    Add support for W3C traceparent propagation.
```

```suggestion
    Add support trace propagation using W3C Trace Context.
    
    The span parenting relationships and sampling decision can be extracted and propagated to instrumented services using the W3C Trace Context standard HTTP header ``traceparent``.
    
    Note: ``tracestate`` HTTP header is not yet supported.
```

---

Sorry, I realized this release note should be more brief.

```suggestion
    Add support trace propagation using W3C Trace Context.  with HTTP header ``traceparent``.
    
    Note: ``tracestate`` HTTP header is not yet supported.
```